From d29b7d6a25934a5ed8ea53bb19a22530fcf1cb81 Mon Sep 17 00:00:00 2001
From: Yao Zi <ziyao@disroot.org>
Date: Wed, 29 Oct 2025 08:17:23 +0000
Subject: [PATCH] configlib: Do not link to unused PkgConfig::XkbFile

No XKB headers are included for configlib, and no Xkb* symbols are
unused in this submodule. Linking to the library seems unnecessary here.

Signed-off-by: Yao Zi <ziyao@disroot.org>
---
 src/lib/configlib/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/configlib/CMakeLists.txt b/src/lib/configlib/CMakeLists.txt
index 6c257c6..1c204d7 100644
--- a/src/lib/configlib/CMakeLists.txt
+++ b/src/lib/configlib/CMakeLists.txt
@@ -19,6 +19,6 @@ target_include_directories(configlib INTERFACE ${CMAKE_CURRENT_SOURCE_DIR} ${CMA
 target_link_libraries(configlib
     Qt${QT_MAJOR_VERSION}::Core Qt${QT_MAJOR_VERSION}::Gui Fcitx5Qt${QT_MAJOR_VERSION}::DBusAddons
     Fcitx5::Core Fcitx5::Utils
-    Fcitx5Qt${QT_MAJOR_VERSION}::WidgetsAddons PkgConfig::XkbFile
+    Fcitx5Qt${QT_MAJOR_VERSION}::WidgetsAddons
 )
 
